media-libs/openjpeg/openjpeg-2.2.0.recipe

```diff
@@ -0,0 +1,89 @@
+SUMMARY="An open-source C-Library for JPEG 2000"
+DESCRIPTION="The OpenJPEG library is an open-source JPEG 2000 codec written \
+in C language. It has been developed in order to promote the use of JPEG \
+2000, the new still-image compression standard from the Joint Photographic \
+Experts Group (JPEG).
+In addition to the basic codec, various other features are under development, \
+among them the JP2 and MJ2 (Motion JPEG 2000) file formats, an indexing tool \
+useful for the JPIP protocol, JPWL-tools for error-resilience, a Java-viewer \
+for j2k-images."
+HOMEPAGE="http://www.openjpeg.org/"
+COPYRIGHT="2002-2014, Communications and Remote Sensing Laboratory, \
+Universite catholique de Louvain (UCL), Belgium
+	2002-2014 Professor Benoit Macq
+ 	2003-2014 Antonin Descampe
+ 	2003-2009 Francois-Olivier Devaux
+ 	2005 Herve Drolon, FreeImage Team
+ 	2002-2003 Yannick Verschueren
+	2001-2003 David Janssens
+	2011-2012, Centre National d'Etudes Spatiales (CNES), France
+	2012, CS Systemes d'Information, France"
+LICENSE="BSD (2-clause)"
+REVISION="1"
+SOURCE_URI="http://github.com/uclouvain/openjpeg/archive/v$portVersion.tar.gz"
+SOURCE_FILENAME="openjpeg-$portVersion.tar.gz"
+CHECKSUM_SHA256="6fddbce5a618e910e03ad00d66e7fcd09cc6ee307ce69932666d54c73b7c6e7b"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	openjpeg$secondaryArchSuffix = $portVersion
+	cmd:opj_decompress$secondaryArchSuffix = $portVersion
+	cmd:opj_compress$secondaryArchSuffix = $portVersion
+	cmd:opj_dump$secondaryArchSuffix = $portVersion
+	lib:libopenjp2$secondaryArchSuffix = 7.0.2 compat >= 7
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libpng16$secondaryArchSuffix
+	lib:libtiff$secondaryArchSuffix >= 5
+	lib:libz$secondaryArchSuffix
+	lib:liblcms2$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	openjpeg${secondaryArchSuffix}_devel = $portVersion
+	devel:libopenjp2$secondaryArchSuffix = 7.0.2 compat >= 7
+	"
+REQUIRES_devel="
+	openjpeg$secondaryArchSuffix == $portVersion
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libpng$secondaryArchSuffix
```

Comment:
IIRC it wouldn't be needed for the devel package? I can change it, no prob (but I think I just left it as it was with the previous recipe) ...